lnclient/lnd/lnd.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 type LNDService struct {
-	client *wrapper.LNDWrapper
-	pubkey string
-	cancel context.CancelFunc
+	client   *wrapper.LNDWrapper
+	nodeInfo *lnclient.NodeInfo
```

Comment:
Lmk if this is good, then I will do this for other backends too

Reply:
I am not sure we should just do this on every backend. If we want to do that, I'd much rather store the network on startup and add a call to get the network to the LNClient interface